jamventProto/jamctrllib/JamCtrlHw.cpp

```diff
@@ -0,0 +1,28 @@
+
+#include "JamCtrlHw.h"
+#include "fmtstr.h"
+
+using namespace std;
+
+
+
+JamCtrlHw::JamCtrlHw()
+{
+
+}
+JamCtrlHw::~JamCtrlHw()
+{
+    
+}
+
+void JamCtrlHw::init() {
```

Comment:
Is this needed? Ok for now.  May want custom init code here.  Otherwise can remove.